### PR TITLE
feat: realtek firmware

### PR DESCRIPTION
**Pull request by linuxmaniac**
fixes https://github.com/siderolabs/pkgs/issues/895

### CODE REVIEW

File: firmware/realtek/pkg.yaml [hunk 1]
```diff
@@ -0,0 +1,23 @@
+name: bnx2-bnx2x
```
Comment (frezbo) [inline firmware/realtek/pkg.yaml:1]:
```suggestion
name: realtek-firmware
```

File: firmware/realtek/manifest.yaml [hunk 2]
```diff
@@ -0,0 +1,10 @@
+version: v1alpha1
+metadata:
+  name: realtek
```
Comment (frezbo) [inline firmware/realtek/manifest.yaml:3]:
```suggestion
  name: realtek-firmware
```

**Comment by linuxmaniac** [comment]
That was fast! Using realtek-firmware as suggested

**Comment by smira** [comment]
it should also be added to `README.md`, and via `.kres.yaml` to the `Makefile`

**Comment by linuxmaniac** [comment]
> it should also be added to `README.md`, and via `.kres.yaml` to the `Makefile`

done

**Review by frezbo — APPROVED** [review]

**Pull request merged**